include/aspect/mesh_deformation/fastscape.h

```diff
@@ -86,6 +87,18 @@ namespace aspect
          */
         void parse_parameters (ParameterHandler &prm) override;
 
+        /**
+         * Enumeration for selecting which type of additional ouput to use in Fastscape vtk.
+         * Select between Fastscape variables.
+         */
+        enum FSScheme
```

Comment:
This class does not have the right name. The name suggests that it has something to do with a FastScape (solver?) scheme, but it controls output. You may want to rename it to reflect what it is used for.